test/unit/spec/api/authSpec.js

```diff
@@ -140,6 +140,8 @@ define(['api/auth', 'narrativeConfig', 'uuid'], (Auth, Config, Uuid) => {
     ];
 
     describe('Test the Auth API module', () => {
+        afterAll(() => TestUtil.clearRuntime());
```

Comment:
This is an idiom that'll show up throughout this PR.

Either adding `TestUtil.clearRuntime` to `afterAll` or `afterEach`, whichever seemed most appropriate.

Reply:
I wish it was possible to have an `afterAll` for every test file. 😖